db/schema.rb

```diff
@@ -909,13 +909,11 @@
     t.text "alert"
     t.boolean "email_locations_admin", default: false
     t.boolean "email_names_admin", default: false
-    t.integer "thumbnail_size", default: 1
     t.integer "image_size", default: 5
     t.string "default_rss_type", limit: 40, default: "all"
     t.integer "votes_anonymous", default: 1
     t.integer "location_format", default: 1
```

Comment:
I suspect the change in the meaning of the default here and in other locations is a major source of the test failures you're seeing.

Reply:
Wow, good catch. I didn't change these intentionally!